### PR TITLE
DNM - Bootstrap tox -edocs entry point

### PR DESCRIPTION
**Pull request by pabelanger**
This is the first commit to update our documentationt to use tox and
sphinx-build to generate documentation.  A follow up patch will enable
documentation builds in Zuul to allow us to gate on doc changes.

Depends-On: https://github.com/ansible/ansible-navigator/pull/104
Signed-off-by: Paul Belanger <pabelanger@redhat.com>

### CODE REVIEW

**Comment by pabelanger** [comment]
Pull-request updated, HEAD is now 67dd1e6e150d05986d062e428fde0424dceef8d6

**Comment by pabelanger** [comment]
Pull-request updated, HEAD is now 22d5308a4d14a4dd0f9add3cd6227793e27bb5c7

**Comment by pabelanger** [comment]
recheck

**Comment by pabelanger** [comment]
Pull-request updated, HEAD is now 8712c2995c29d0fec07729d262248f703eafc8b9

**Comment by pabelanger** [comment]
Pull-request updated, HEAD is now 943c1cb524347da1e058b75c3ec425b803f9f6f2

**Comment by pabelanger** [comment]
Pull-request updated, HEAD is now 995700bb804e0b739cd826cd242250af39eb7b51

**Comment by pabelanger** [comment]
Pull-request updated, HEAD is now 81d997e13b581a9fed17f93264cd8b18afb8dc0b

**Comment by pabelanger** [comment]
Pull-request updated, HEAD is now 20609340c99652586fc0abea42556399a7b2e5e3

File: setup.cfg [hunk 1]
```diff
@@ -43,7 +43,7 @@ install_requires =
     dataclasses; python_version < "3.7.0"
 python_requires = >=3.6.1
 
-[options.data_files]
+[data_files]
```
Comment (webknjaz) [inline setup.cfg:46]: This change is wrong. All the sections with options must be prefixed with `options.` for setuptools to find them.

File: tox.ini [hunk 2]
```diff
@@ -31,6 +31,11 @@ max-line-length = 160
 builtins = _
 exclude = .git,.tox
 
+[testenv:docs]
+deps = -r{toxinidir}/doc/requirements.txt
+commands =
+  sphinx-build -E -W -d doc/build/doctrees -b html doc/source/ doc/build/html
```
Comment (webknjaz) [inline tox.ini:37]: Maybe make it strict?
```suggestion
  sphinx-build -E -W --keep-going -n -d doc/build/doctrees -b html doc/source/ doc/build/html
```

File: doc/source/conf.py [hunk 3]
```diff
@@ -0,0 +1,52 @@
...
+# This pattern also affects html_static_path and html_extra_path.
+exclude_patterns = []
+
+
+# -- Options for HTML output -------------------------------------------------
+
+# The theme to use for HTML and HTML Help pages.  See the documentation for
+# a list of builtin themes.
+#
+html_theme = 'alabaster'
```
Comment (webknjaz) [inline doc/source/conf.py:47]: Try https://github.com/ansible-community/sphinx_ansible_theme or https://github.com/pradyunsg/furo maybe?

**Comment by cidrblock** [comment]
closing for now as discussed

**Pull request closed without merging**